src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -12,6 +12,10 @@
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry\Internal\UnconditionalSuppressMessageAttribute.cs" LinkBase ="Common" Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'" />
+  </ItemGroup>
+
```

Comment:
```suggestion
```

You don't need to define these attributes in all the assemblies. Just in the assemblies that need to use them.